frontend/src/components/involvement/get_involved.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import './get_involved.css';
+// import icon from 
+
+function GetInvolved() {
+  const arrayCards = [
+    [
+      'Report Issues',
+      'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eligendi, magni.'
+    ], [
+      'Code Fixes', 'If you came across something that isn\'t working perfectly, feel free to open up an issue for the same'
+    ], [
+      'Improve Documentation',
+      'If you came across something that isn\'t working perfectly, feel free to open up an issue for the same'
+    ], [
+      'Propose Ideas', 'If you came across something that isn\'t working perfectly, feel free to open up an issue for the same'
+    ], [
+      'Improve Documentation',
+      'If you came across something that isn\'t working perfectly, feel free to open up an issue for the same'
+    ], [
+      'Improve Documentation',
+      'If you came across something that isn\'t working perfectly, feel free to open up an issue for the same'
+    ]
+  ]
+  return (
+    <main>
+      <div id="hero">
+        <div id="heading">
+          <div id="title">GET INVOLVED!</div>
+        </div>
+      </div>
+      <div id="allCards">
+        {arrayCards.map(element => {
+          return (
+            <div className="card-item">
```

Comment:
Pass a key parameter to this div.
Use the 2nd parameter of the map function as a key for the first div that mounted all other tags inside the map function.
ex: -  key={ index}    'index;   is the 2nd parameter.